### DISCUSSION TITLE
Repo Insights

### DISCUSSION THREAD

**Discussion by pete4abw** [community/community#3650] [Mobile]
Insight stats are missing from mobile app. W/B nice to have.

**Comment by brianlovin** [comment]
Thanks for the note @pete4abw – this isn't prioritized on the roadmap yet, but we'll keep an eye out for more requests for this feature. What insights stats are you trying to access on the go? What would be the most useful place for us to start?

**Reply by pete4abw** [reply]
@brianlovin Well, Insights/traffic, pulse, in particular. In addition having access to Wiki would be useful for R/W access. I just like to see consistency between the Desktop and the App. Thx.
